Sources/Bot/API/Response.swift

```diff
@@ -0,0 +1,66 @@
+
+// Copied from https://github.com/mdiep/Tentacle
+
+import Foundation
+
+let LinksRegex = try! NSRegularExpression(pattern: "(?<=\\A|,) *<([^>]+)>( *; *\\w+ *= *\"[^\"]+\")* *(?=\\z|,)")
+let LinkParamRegex = try! NSRegularExpression(pattern: "; *(\\w+) *= *\"([^\"]+)\"")
```

Comment:
I brought this from [Tentacle](https://github.com/mdiep/Tentacle) so I was trying to keep it unmodified but I'm actually changing this to remove the stuff we don't need and structure to fit our purpose.

Reply:
Just curious: why don't we depend on Tentacle directly instead is reinventing the wheel? NIH?

Reply:
I tried @AliSoftware but our models are slight different, mainly `PullRequest` which only contains the url links for the pull request, diff, ... 

Also another problem I found out was with extending it, `Resource` does not have a public initialiser so we can't extend to other APIs that we need like fetching all pull requests from a repository.

Based on the structure we are using I think we can migrate to it if we address this limitations but I didn't want to wait more time on this so it was faster to develop our own infra-structure.

Reply:
Ok. I think it's worth opening an issue on the repo at least to ask for those changes and making the init public. Will ping the author in Slack too 😉

Reply:
👍 I was planning to open a ticket anyway but I'd like to get this one done first 😅

Reply:
@dmcrodrigues I'm happy to add you as a contributor to Tentacle too if you're interested in making improvements. :)

Reply:
@mdiep I'm definitely interested ❤️  I didn't take that route from the beginning because I was short on time but I based most of our implementation to ensure we could switch in the future.